addons/pos_restaurant/static/src/overrides/models/models.js

```diff
@@ -29,7 +30,9 @@ patch(Order.prototype, {
         super.init_from_JSON(...arguments);
         if (this.pos.config.module_pos_restaurant) {
             this.tableId = json.table_id;
-            this.validation_date = moment.utc(json.creation_date).local().toDate();
+            const format = "yyyy-MM-dd HH:mm:ssZZ";
+            json.creation_date = json.creation_date || DateTime.now().toFormat(format); //ex : json.creation_date = "2023-08-25 13:24:16+02:00"
+            this.validation_date = DateTime.fromFormat(json.creation_date, format).toJSDate();
```

Comment:
I do not understand why you need to replace `json.creation_date` here
We will discuss this IRL tomorrow.

Reply:
Cause luxon do not interpolate `undefined` as input where moment(undefined) is simply moment(); (now)